src/components/Navbar/Navbar.js

```diff
@@ -5,14 +5,17 @@ import { useWishList } from '../../context/wishlist-context'
 import { useCart } from '../../context/cart-context'
 export const Navbar = () => {
     const { auth, setAuth } = useAuth()
-    const { wishlist } = useWishList()
-    const { cart } = useCart()
+    const { wishlist, setWishlist } = useWishList()
+    const { cart, setCart } = useCart()
     const navigate = useNavigate()
     const logoutHandler = () => {
         setAuth({ token: '', isAuthenticated: false })
         localStorage.removeItem('token')
         localStorage.removeItem('isAuthenticated')
```

Comment:
only token can check whether authenticated or not..better to not to put extra property for that